src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/SeedColumns.kt

```diff
@@ -12,6 +12,10 @@ data class SeedColumns(
 
   fun uuidOrNull(label: String) = columns[label]?.let { UUID.fromString(it.trim()) }
 
+  fun intOrNull(label: String) = columns[label]?.toIntOrNull()
+
+  fun longOrNull(label: String) = columns[label]?.toLongOrNull()
```

Comment:
The code `columns [label]? Used repeatedly so we could extract it into a function and use it instead.

Something like `    private fun getTrimmedStringValue(label: String): String? = columns[label]?.trim()`

Reply:
Also, all these functions return something, so it could be getXXX().